src/utils/call_openai_summarise.py

```diff
@@ -18,7 +18,7 @@ def create_openai_summary(
         ]
         completion = client.chat.completions.create(
             messages=messages,  # type: ignore
-            max_tokens=250,
+            max_tokens=1000,
```

Comment:
If we're upping this to 1k should be include something in the prompt that tries to ensure that it's as concise as possible? (Might already exist). Also are we happy with a temp of .2 when summarising 100 records?

Reply:
I think it couldn't hurt. I think we should be programmatic about it to ensure the summaries are useful - we could do some experimentation with prompts to ensure we get concise but meaningful summaries.